apps/ledger-live-mobile/src/colors.tsx

```diff
@@ -71,6 +71,8 @@ export const lightTheme = {
     translucentGrey: "rgba(153, 153, 153, 0.2)",
     lightLiveBg: "#e3dfff",
 
+    darkGrey: "#333333",
```

Comment:
Same as `theme.colors.opacityDefault.c80` (AKA 0, 0, 0, 0.8) so unless it's supposed to be resistant to light/dark theme changes then this might just be badly named in the design?

Reply:
Was using the `theme` rules that has been manually defined here https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/colors.tsx.

Since I now see that the default one from styled-components is been used elsewhere, I'll replace it with that one.